smartpower3/screen.cpp

```diff
@@ -925,7 +925,7 @@ void Screen::fsInit(void)
 	log_interval = NVS.getInt("log_interval");
 
 	if (NVS.getString("wifi_conn_ok") != "true")
-		wifiManager->state = 1;
+		wifiManager->credentials_state = STATE_CREDENTIALS_INVALID;
```

Comment:
ints used like this are really hard to understand, which leads to errors - enums are better. I just hope that I've interpreted the intention in the name correctly.